machine-learning/lambda-v2/README.md

```diff
@@ -2,14 +2,20 @@
 
 These example components are used to run sample image classification and object detection inferences with AWS lambda.
 
+
 Image Classification 
- - com.lambda.DLRImageClassification 
+ - com.lambda.DLRImageClassification(No container  mode)
+ - com.lambda.Container.DLRImageClassification(Greengrass container mode)
```

Comment:
Should the component name be `com.lambda.container.DLRImageClassification` to align with our standard to lowercase namespaces in the component name? (Note the lowercase `container`).